app/Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+flask = "*"
+flask-sqlalchemy = "*"
+
+[requires]
+python_version = "3.7"
```

Comment:
might want to change this later on to make sure everyone has the same python version